config/locales/en.yml

```diff
@@ -62,6 +62,8 @@ en:
               other: Unsupported MIME types detected in %{file_names}.
             multiple_channel_integrations: Specifying multiple channel integrations
               in requirements.json is not supported.
+            too_many_oauth_parameters: "Too many parameters with type 'oauth': one permitted"
```

Comment:
This doesn't appear to be referenced below.

Reply:
Logan, yes... this was a rebase issue. I removed this line 👍